test/test_fx.py

```diff
@@ -186,6 +189,142 @@ def test_resnet(self):
+            #       interpreter's instruction format ======
+            instructions = []
+            constant_idx = 0
+            constants = {}
+            fn_input_names = []
+
+            target_to_name = {
+                operator.add : "add",
+                operator.mul : "mul"
+            }
+
+            # For each instruction, create a triple
+            # (instruction_name : str, inputs : List[str], output : str)
+            # to feed into the C++ interpreter
+            for n in mod.graph.nodes:
+                target, args, out_name = n.target, n.args, n.name
+                assert len(n.kwargs) == 0, "kwargs currently not supported"
+
+                if n.op == 'placeholder':
+                    # Placeholders specify function argument names. Save these
+                    # for later when we generate the wrapper GraphModule
+                    fn_input_names.append(target)
+                elif n.op == 'call_function':
+                    assert target in target_to_name, "Unsupported call target " + target
+                    arg_names = []
+                    for arg in args:
+                        if not isinstance(arg, Node):
+                            # Pull out constants. These constants will later be
+                            # fed to the interpreter C++ object via add_constant()
+                            arg_name = f'constant_{constant_idx}'
+                            constants[arg_name] = torch.Tensor(
+                                [arg] if isinstance(arg, numbers.Number) else arg)
+                            arg_names.append(arg_name)
+                            constant_idx += 1
+                        else:
+                            arg_names.append(arg.name)
+                    instructions.append((target_to_name[target], arg_names, out_name))
+
+                else:
+                    raise RuntimeError('Unsupported opcode' + n.op)
+
+            interpreter = torch.classes._TorchScriptTesting._ElementwiseInterpreter()
+            # Load constants
+            for k, v in constants.items():
+                interpreter.add_constant(k, v)
+            # Specify names for positional input arguments
+            interpreter.set_input_names(fn_input_names)
+            # Load instructions
+            interpreter.set_instructions(instructions)
+            # Specify name for single output
+            interpreter.set_output_name(mod.graph.result.name)
+
+            # ===== Stage 3: Create a wrapper GraphModule around the interpreter =====
+            class WrapperModule(torch.nn.Module):
+                def __init__(self, interpreter):
+                    super().__init__()
+                    self.interpreter = interpreter
+
+            wrapper = WrapperModule(interpreter)
+
+            # Create a graph that: 1) Takes function arguments 2) Invokes the interpreter
+            # 3) Returns the speficied return value
+
```

Comment:
`symbolic_trace(module, <delegate_that_considers Wrapper a root module>)` would do the same with a lot less code.

Reply:
Yeah that would be ideal, but I can't get `__call__` exposed on TorchBind classes without breaking `hasattr`. I can add a TODO